### PR TITLE
DM-33143: first release

### PR DESCRIPTION
**Pull request by r-owen**
Get Jenkins running (this should just work, now that @tribeiro has made labjack-ljm part of our standard environment).
Restore the ability to communicate via USB (by reverting the previous ticket), since our installation supports that.

### CODE REVIEW

**Comment by tribeiro** [comment]
I think the Jenkinsfile still need some work for the tests to run and pass... It is currently doing only [this](https://github.com/lsst-ts/ts_ess_labjack/blob/2a1291221a1ebb871826b52fe9cc330cb2d0fa2b/Jenkinsfile#L59-L62):

```
cd /home/saluser/repos/ts_ess_common
/home/saluser/.checkout_repo.sh ${work_branches}
git pull
make_idl_files.py ESS
```

Note that ts_ess_common is not part of the development environment.

**Comment by wvreeven** [comment]
I agree with @tribeiro. The build now fails because `/home/saluser/repos/ts_ess_common` doesn't exist. The ts_ess_csc Jenkinsfile contains the correct way of checking out ts_ess_common.

**Comment by r-owen** [comment]
Thank you for the diagnosis of the Jenkins problem. I fixed that, as well as an error in setup.py and Jenkins is now passing (though it does not seem to be properly connected to this PR). https://tssw-ci.lsst.org/view/LSST_TandS/job/LSST_Telescope-and-Site/job/ts_ess_labjack/view/change-requests/job/PR-3/

File: Jenkinsfile [hunk 1]
```diff
@@ -56,7 +56,12 @@ pipeline {
                         cd /home/saluser/repos/ts_idl
                         /home/saluser/.checkout_repo.sh ${work_branches}
                         git pull
+                        cd /home/saluser/repos
+                        rm -rf ts_ess_common  # In case it becomes part of repos
```
Comment (wvreeven) [inline Jenkinsfile:60]: I strongly recommend removing this line. If ts_ess_common becomes part of `repos` then the build will fail which will be a sign to adjust Jenkinsfile to the new situation. If this line remains then that will go unnoticed.

**Review by wvreeven — CHANGES_REQUESTED** [review]
This all looks great, except the one line in Jenkinsfile where a potentially present `ts_ess_common` directory gets removed. See my review comment there. Please explain to me if you have a good reason to leave that line in but otherwise please remove it.

**Comment by wvreeven** [comment]
And the Jenkins build is visible in this PR page. Not sure why you didn't see it before.

File: Jenkinsfile (hunk 1 above)
Comment (r-owen) [reply Jenkinsfile]: OK.

**Comment by r-owen** [comment]
I made the change you requested.

The problem with Jenkins is that pushing change doesn't trigger a new Jenkins job. If I trigger the jobs manually then the status does show up.

**Review by wvreeven — APPROVED** [review]
Looks good, thanks for the changes.

**Pull request merged**